game/requesthandler.cc

```diff
@@ -3,24 +3,16 @@
 
 #ifdef USE_WEBSERVER
 
-// RequestHandler::RequestHandler(Songs& songs):m_songs(songs),
-// m_restinio_server {
-//         	restinio::own_io_context(),
-//         	[&](auto& settings) {
-//         		settings
-//                 .port( port )
-//                 .address( url )
-//                 .separate_accept_and_create_connect(true)
-//                 .request_handler(std::make_unique<Performous_Router_t>());
-//                 } {}
-
 RequestHandler::RequestHandler(std::string url, unsigned short port, Songs& songs):m_listener("http://" + url), m_songs(songs), m_restinio_server(restinio::own_io_context(), make_server_settings(url, port)) {
+	m_local_ip = RequestHandler::getLocalIP(std::string("1.1.1.1"));
```

Comment:
Hmmm what is this ip address?

Reply:
CloudFlare's DNS. It's what we were already using.

Reply:
Alright 👍 now i remember we needed a 3rd party so we can get our ip address